what-ive-learned-from-inheriting-a-popular-open-source-project.md

```diff
@@ -0,0 +1,364 @@
+<!--
+headline: What I&#x27;ve Learned From Inheriting A Popular Open Source Project
+description: a summary of some of the things I&#x27;ve learned from taking over development of the yargs command-line-parser.
+author: Ben Coe
+datePublished: 2015-04-21
+twitter: benjamincoe
+github: bcoe
+-->
+
+_this story begins all the way back in 2014, shortly after I began work at npm_
```

Comment:
I like the use of the word "story" and that you provide context within which said story takes place